package.json

```diff
@@ -33,8 +33,7 @@
         "lodash.clonedeep": "^4.5.0",
         "lodash.debounce": "^4.0.8",
         "lodash.throttle": "^4.1.1",
-        "nocache": "^2.0.0",
-        "openfin-launcher": "^1.3.11"
```

Comment:
The openfin-launcher probably shouldn't have been a production dependency because it is only used in development, but yes the seed no longer has any dependency on openfin-launcher. When I spoke with OF about the issue they said they were moving away from openfin-launcher in favor of the hadouken-js-adapter.